.github/workflows/staging-deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - 'main'
+      - '**-staging-test'
```

Comment:
intentially leaving this here to allow us to trigger a stagging release for non-merged branches as long as we name with the `-staging-test` suffix, e.g. `acb/my-test-branch-staging-test`